src/components/Table/hoc/withTableSettings/withTableSettings.tsx

```diff
@@ -114,7 +114,7 @@ function prepareUpdateSettings(items: TableColumnSetupItem[]): TableSettingsData
 export interface WithTableSettingsProps {
     settingsPopupWidth?: string;
     settings: TableSettingsData;
-    updateSettings: (data: TableSettingsData) => Promise<void>;
+    updateSettings: (data: TableSettingsData) => void;
```

Comment:
Is `() => Promise<void>` assignable to `() => void`?

Reply:
Yes